src/web/components/SectionTitle.tsx

```diff
@@ -31,9 +31,9 @@ const headerStyles = css`
     }
 `;
 
-const descriptionStyles = css`
+const descriptionStyles = (fontColour?: string) => css`
```

Comment:
Is there a type for palette text ?

Reply:
You mean a type from Source? That would be cool. @SiAdcock ?

Reply:
I had a chat with Simon about this and he's going to create a ticket for this